src/Client/Settings.java

```diff
@@ -84,6 +84,7 @@
   public static HashMap<String, Integer> NAME_PATCH_TYPE = new HashMap<String, Integer>();
   public static HashMap<String, Integer> COMMAND_PATCH_TYPE = new HashMap<String, Integer>();
   public static HashMap<String, Boolean> ATTACK_ALWAYS_LEFT_CLICK = new HashMap<String, Boolean>();
+  public static HashMap<String, Boolean> KEEP_MAGIC_PRAYER_TAB_CURSOR = new HashMap<String, Boolean>();
```

Comment:
refactor to KEEP_SCROLLBAR_POS_MAGIC_PRAYER, or similar, with "SCROLLBAR"

CURSOR is inappropriate, does not refer to anything related to the Mouse Cursor.